ReadmeSrc/Rows.js

```diff
@@ -0,0 +1,69 @@
+const headers = [
+  'Package',
+  //   'Version',
```

Comment:
this is the version of the package used in example (or isn't it?). I would still show it. And also show latest version in npm - to be able to detect how given example is outdated.